### PR TITLE
kubecost: Bump kubecost to v1.74.0

### PR DESCRIPTION
**Pull request by gracedo**
**What type of PR is this?**

chore/bug

**What this PR does/ why we need it**:

Bumps kubecost to latest version which includes a [fix](https://github.com/kubecost/cost-analyzer-helm-chart/pull/765) for the previous v1.73.0 version which introduced a change to always deploy a psp which caused collision issues in kommander. 

**Which issue(s) this PR fixes**:

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
fix(kubecost): Disable PSP which causes name collisions if multiple instances of Kubecost are running in a cluster
```

**Checklist**

* [x] *If a chart is changed, the chart version is correctly incremented.*
* [x] The commit message explains the changes and why are needed.
* [ ] The code builds and passes lint/style checks locally.
* [ ] The relevant subset of integration tests pass locally.
* [ ] The core changes are covered by tests.
* [ ] The documentation is updated where needed.

### CODE REVIEW

**Review by hectorj2f — APPROVED** [review]

File: stable/kubecost/values.yaml [hunk 1]
```diff
@@ -57,6 +57,9 @@ cost-analyzer:
     enabled: true
     # storageClass: "-"
 
+  podSecurityPolicy:
+    enabled: false
```
Comment (hectorj2f) [inline stable/kubecost/values.yaml:61]: Do we want that ? Was it the default behavior before ?

Comment (hectorj2f) [reply stable/kubecost/values.yaml:61]: The default for kubecost users is true :/. 

Should we disable it for the main mgmt or better use kubefed overrides to disable it in the federated Addon kubecost when placed in the mgmt cluster (which we might have a label for 🤔 ) ?

Comment (gracedo) [reply stable/kubecost/values.yaml:61]: It was defaulted to false in the past, the latest releases changed the default to true which caused this to break in kommander. I think maybe the correct fix is to use a unique name for PSP so there won't be name conflicts, I can try and put up a pr against kubecost

Comment (hectorj2f) [reply stable/kubecost/values.yaml:61]: okay, sounds good. Let's track it in a jira ticket. It sounds good for me disabling now.

Comment (gracedo) [reply stable/kubecost/values.yaml:61]: https://jira.d2iq.com/browse/D2IQ-74182 will open a pr upstream

Comment (hectorj2f) [reply stable/kubecost/values.yaml:61]: Great @gracedo :).

Comment (gracedo) [reply stable/kubecost/values.yaml:61]: https://github.com/kubecost/cost-analyzer-helm-chart/pull/773

**Comment by gracedo** [comment]
There are a couple of patches on 0.13.x for kommander 1.3/konvoy 1.7, let's get those in first before bumping the version here

**Review by makkes — APPROVED** [review]

**Review by hectorj2f — APPROVED** [review]

**Review by joejulian — APPROVED** [review]

**Review by branden — APPROVED** [review]

**Pull request merged**